libraries/AP_Compass/AP_Compass.h

```diff
@@ -336,6 +333,21 @@ friend class AP_Compass_Backend;
         return (uint16_t)_offset_max.get();
     }
 
+    // return maximum allowed angle difference (in degrees) between compass' in xyz plane
+    AP_Float get_ang_diff_xyz_max(void) const {
+        return _ang_diff_xyz_max;
+    }
+
+    // return maximum allowed angle difference (in degrees) between compass' in xy plane
+    AP_Float get_ang_diff_xy_max(void) const {
+        return _ang_diff_xy_max;
+    }
+
+    // return maximum allowed mag field length difference (in mG) between compass' in xy plane
+    AP_Float get_len_diff_xy_max(void) const {
+        return _len_diff_xy_max;
+    }
```

Comment:
With C++, the MATE team has adopted the convention that "get" and "set" functions use the same overloaded function name:
```
AP_Float ang_diff_xyz_max(void);          // get function
void ang_diff_xyz_max(AP_Float ang_max);  // set function
```
Should we be applying that convention here as well to have that consistency with MATE?
(@zalice @adriennet823 @AydenChubbic @bhiln @viet-nguyen-matternet if you care to chime on this)

Reply:
If there is no existing nomenclature, I think it makes sense to use the same naming convention across as much of our code base as possible

Reply:
Good catch on that @rabad-matternet.

More broadly, I'm requesting that the accessor functions be removed since no class other than the this class needs to access this data. If we need it later, we'll add accessors.